src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/IepLevelService.kt

```diff
@@ -1,25 +1,18 @@
 package uk.gov.justice.digital.hmpps.incentivesapi.service
 
-import kotlinx.coroutines.flow.filter
 import kotlinx.coroutines.flow.toList
 import org.springframework.stereotype.Service
 import org.springframework.transaction.annotation.Transactional
 import uk.gov.justice.digital.hmpps.incentivesapi.dto.prisonapi.IepLevel
-import uk.gov.justice.digital.hmpps.incentivesapi.jpa.repository.IepLevelRepository
 
 @Service
 class IepLevelService(
-  private val iepLevelRepository: IepLevelRepository,
   private val prisonApiService: PrisonApiService,
 ) {
-
   @Transactional(readOnly = true)
   suspend fun getIepLevelsForPrison(prisonId: String, useClientCredentials: Boolean = false): List<IepLevel> {
-    val iepLevelMap = iepLevelRepository.findAll().toList().associateBy { it.iepCode }
 
-    return prisonApiService.getIepLevelsForPrison(prisonId, useClientCredentials)
-      .filter { iepLevelMap[it.iepLevel]?.active == true }
```

Comment:
Are we removing this because there’s a scenario where our `iep_level` table will diverge from what configured in NOMIS and we’ll be incorrectly filtering records out?